schema/data_feed/tracker.fbs

```diff
@@ -58,4 +61,10 @@ table TrackerInfo {
 
     /// name to display as the name of the tracker set by the user
     custom_name: string;
+    
+    // Reference adjustment parameters for IMU trackers (full reset, yaw, mounting)
+    ref_adj_gyro_fix: solarxr_protocol.datatypes.math.Quat;
```

Comment:
"gyro fix" isn't really descriptive to me - IDK what a fix in this context is. Goal is for someone reading this schema to immediately know what it means.

Could you add a more descriptive comment (with `///`) to each of these fields?